File: src/main/java/org/pulcifer/climate/model/Climate.java
package org.pulcifer.climate.model;

import com.fasterxml.jackson.annotation.JsonInclude;

import java.math.BigDecimal;
import java.util.List;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class Climate {
    private String raintype;
    private String raindef;
    private String rainunit;
    private BigDecimal datab;
    private BigDecimal datae;
    private BigDecimal tempb;
    private BigDecimal tempe;
    private BigDecimal rdayb;
    private BigDecimal rdaye;
    private BigDecimal rainfallb;
    private BigDecimal rainfalle;
    private String climatefromclino;
    private List<ClimateMonth> climateMonth;

    public String getRaintype() {return this.raintype;}
    public String getRaindef() {return this.raindef;}
    public String getRainunit() {return this.rainunit;}
    public BigDecimal getDatab() {return this.datab;}
    public BigDecimal getDatae() {return this.datae;}
    public BigDecimal getTempb() {return this.tempb;}
    public BigDecimal getTempe() {return this.tempe;}
    public BigDecimal getRdayb() {return this.rdayb;}
    public BigDecimal getRdaye() {return this.rdaye;}
    public BigDecimal getRainfallb() {return this.rainfallb;}
    public BigDecimal getRainfalle() {return this.rainfalle;}
    public String getClimatefromclino() {return this.climatefromclino;}
    public List<ClimateMonth> getClimateMonth() {return this.climateMonth;}

    public void setClimateMonth(List<ClimateMonth> climateMonth) {
        this.climateMonth = climateMonth;
    }
}

File: src/main/kotlin/org/pulcifer/climate/model/Climate.kt
package org.pulcifer.climate.model

import com.fasterxml.jackson.annotation.JsonInclude
import java.math.BigDecimal

@JsonInclude(JsonInclude.Include.NON_NULL)
class Climate {
    var raintype: String? = null
    var raindef: String? = null
    var rainunit: String? = null
    var datab: BigDecimal? = null
    var datae: BigDecimal? = null
    var tempb: BigDecimal? = null
    var tempe: BigDecimal? = null
    var rdayb: BigDecimal? = null
    var rdaye: BigDecimal? = null
    var rainfallb: BigDecimal? = null
    var rainfalle: BigDecimal? = null
    var climatefromclino: String? = null
    var climateMonth: List<ClimateMonth?>? = null
}